### ISSUE TITLE
Shadow in text or images

### ISSUE THREAD

**Issue by Carleslc** [andymeneely/squib#369]
Is there any shadow option available to display text with a drop shadow? I don't see any in [text DSL](https://squib.readthedocs.io/en/latest/dsl/text.html). Or any alternative in ruby to create this behaviour.

Sometimes you want a text to have some shadow, perhaps to differentiate it better from the background. Creating a custom designed image for that defeats the purpose of this library.

The same applies to images. I see there is some drop shadow feature for the whole card in save_png, but not for specific images.

**Comment by andymeneely** [comment]
No, I don't have one at the moment - would be relatively easy to do though. I'll add it as a feature idea!
